Makefile

```diff
@@ -36,7 +36,7 @@ BUILD_CONTAINER := docker-root-built
 
 BUILT := `docker ps -aq -f name=$(BUILD_CONTAINER) -f exited=0`
 STR_CREATED := $$(docker inspect -f '{{.Created}}' $(BUILD_IMAGE) 2>/dev/null)
-IMG_CREATED := `date -j -u -f "%FT%T" "$(STR_CREATED)" +"%s" 2>/dev/null || echo 0`
+IMG_CREATED := $(shell scripts/stamptounix $(STR_CREATED))
 
```

Comment:
$(shell ...) is evaluated just once on starting `make`, but it should be evaluated on runtime at each tasks.
That's why I use `` format instead of $().